### ISSUE TITLE
Provide Details About AllowLimitedAccess Value

### ISSUE THREAD

**Issue by aakash-shah** [MicrosoftDocs/office-docs-powershell#7742]
Under the `-ConditionalAccessPolicy` parameter, one of the values is `AllowLimitedAccess`. However, this Set-SPOSite page does not contain details or an explanation of what specifically `AllowLimitedAccess` restricts or allows.

The Set-SPOSite page references back to [Control access from unmanaged devices](https://docs.microsoft.com/en-us/sharepoint/control-access-from-unmanaged-devices). However the "Control access from unmanaged devices" page also does not define what `AllowLimitedAccess` does and links back to this Set-SPOSite page creating a circular loop.

Please provide details about what the `AllowLimitedAccess` value provided to the `-ConditionalAccessPolicy` parameter does.

Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2c62bf4a-ac47-96ce-33fb-0d5ea9b178ae
* Version Independent ID: 301586d3-ec04-c42d-4681-0bb59443d35c
* Content: [Set-SPOSite (SharePointOnlinePowerShell)](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/set-sposite?view=sharepoint-ps)
* Content Source: [sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md)
* Service: **sharepoint-powershell**
* GitHub Login: @trent-green
* Microsoft Alias: **trgreen**

**Comment by get-itips** [comment]
Hello @aakash-shah how are you? Hope you are doing good!
The meaning of `AllowLimitedAccess` is indeed in the linked page:

> Limiting access allows users to remain productive while addressing the risk of accidental data loss on unmanaged devices. When you limit access, users on managed devices will have full access (unless they use one of the browser and operating system combinations listed below). Users on unmanaged devices will have browser-only access with no ability to download, print, or sync files. They also won't be able to access content through apps, including the Microsoft Office desktop apps. When you limit access, you can choose to allow or block editing files in the browser. When web access is limited, users will see the following message at the top of sites.

Hope this information helps you

**Comment by aakash-shah** [comment]
Hello! Thanks for the reply and for the information! You are correct that this is included on the page. However, the issue I found is when I was at Set-SPOSite, I then linked to "Control access from unmanaged devices" and did a search for "AllowLimitedAccess", and the section that had this information did not help clarify what specifically this did. Perhaps a table that includes a comparison of what these 3 parameter values allow and disallow would be helpful using the specific parameter value names themselves. 

Also on the "Control access from unmanaged devices" page, if you click on "Limit Access" in the table of contents, it skips past the section you referenced that has this information.

This page can be improved by ensuring that if I do a search for AllowLimitedAccess, I am able to find specific details about what this allows and disallows, and how it compares to the other 2 parameter values.

Thanks!
